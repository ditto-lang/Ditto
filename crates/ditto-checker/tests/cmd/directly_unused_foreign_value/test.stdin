module Test exports (..)

foreign unused_impl: (Int) -> Bool
