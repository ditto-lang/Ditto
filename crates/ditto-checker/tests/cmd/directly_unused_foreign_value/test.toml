bin.name = "ditto-checker-testbin-check-module"
args = ["directly_unused_foreign_value.ditto"]
fs.sandbox = true
