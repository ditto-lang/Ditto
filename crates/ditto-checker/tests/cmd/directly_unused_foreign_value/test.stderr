
  ⚠ unused foreign value
   ╭─[directly_unused_foreign_value.ditto:2:1]
 2 │ 
 3 │ foreign unused_impl: (Int) -> Bool;
   · ─────────────────┬─────────────────
   ·                  ╰── this isn't being used
   ╰────

